app/imports/schemas/TestForm.js

```diff
@@ -1,23 +1,40 @@
-export const FormSchema = new SimpleSchema({
+/* globals _ */
+import { Upload } from '/imports/schemas/Upload.js';
+
+const Schema = new SimpleSchema({
   name: {
     type: String,
   },
 
-  category: {
-    type: String,
-    allowedValues: [
-      'news',
-      'image',
-      'video',
-    ],
+  idCard: {
+    type: Upload,
+    // This makes the field 'required'
+    autoValue() {
+      if (_.isEmpty(this.value.file)) {
+        this.unset();
+      }
+    },
   },
 
-  publishedDate: {
-    type: Date,
+  cv: {
+    type: Upload,
   },
 
-  published: {
-    type: Boolean,
-    optional: true,
+  photos: {
+    type: [ Upload ],
+    maxCount: 2,
   },
+  
+  docs: {
+    type: [ Upload ],
+    // This makes minCount work
+    autoValue() {
+      return _.filter(this.value, (val) => !_.isEmpty(val.file));
+    },
+    minCount: 1,
+    maxCount: 2,
```

Comment:
When you have a multiple uploads field (`[Upload]`) and you require at least one then you use `minCount: 1`. That is just SimpleSchema basics.

The code for this files is like this
idCard: single upload, required
cv: single upload, optional
photos: multiple uploads, no min, max 2
docs: multiple uploads, min 1, max 2

`autoValue` has to be declared in order to enforce these conditions.